src/helix/endpoints/clips/create_clip.rs

```diff
@@ -12,9 +12,9 @@
 //! let request = create_clip::CreateClipRequest::broadcaster_id("1234");
 //! ```
 //!
-//! ## Response: [CreatedClip]
+//! ## Response: [Vec<CreatedClip>]
```

Comment:
```suggestion
//! ## Response: [CreatedClip]
```

We always link to the item that is returned. Moreover, `[Vec<CreatedClip>]` is invalid markdown (rustdoc also complains about this).